app/src/main/java/org/gdg/frisbee/android/arrow/ArrowTaggedActivity.java

```diff
@@ -74,22 +80,15 @@ protected String getTrackedViewName() {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_arrow_tagged);
-
-        adapter = new OrganizerAdapter(this, 0);
+        getWindow().setBackgroundDrawable(new ColorDrawable(getResources().getColor(R.color.recycler_window_bg)));
```

Comment:
Can you set this in the theme of the Activity. When we do it like this it would be really hard to find for other contributors to find this line when needed.